rclrs/src/rcl_bindings.rs

```diff
@@ -3,6 +3,7 @@
 #![allow(non_upper_case_globals)]
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
+#![allow(improper_ctypes)]
```

Comment:
I think it might be a good idea to separate this not allow from the rest of the blocks and add a comment referencing that its coming from ROS 2's new runtime reflection feature in Iron and Rolling. I can see this list getting quite long and goofy over time

Reply:
Done in 80bb0acdad8aec8fae60130e5ac62765c69fbc19

Reply:
I had to fix the formatting because `rustfmt` was complaining.